Bit-Manipulation/test/CountSetBits.test.js

```diff
@@ -0,0 +1,22 @@
+import { CountSetBits } from '../CountSetBits'
```

Comment:
I was more thinking something like:

```javascript
test("CountSetBits", () => {
    const tests = {
        // binary representation: set bits
        '1001': 2,
        '111': 3,
    }
    for (binary in tests)
         expect(CountSetBits(parseInt(binary, 2))).toBe(tests[binary])
})

```

Reply:
I did not use quoted for keys because of:
![image](https://user-images.githubusercontent.com/63806909/192274721-c74c4d86-4cc2-410a-be1f-bbce45cb6ec7.png)

Reply:
I did not intend a reversal to the more verbose form. You can even use a list of lists if you like - all you need is a convenient way to test that a certain input leads to a certain output without copy-pasting. Here's another alternative:

```javascript
const testCountSetBits = (input, expected_output) => expect(CountSetBits(input)).toBe(expected_output)
test('CountSetBits', () => {
    testCountSetBits(25, 3)
    testCountSetBits(36, 2)
})
```
etc., or, IMO preferable since it makes the binary representation apparent:

```javascript
const testCountSetBits = (input_bin_str, expected_output) => expect(CountSetBits(parseInt(input, 2))).toBe(expected_output)
test('CountSetBits', () => {
    testCountSetBits("0", 0)
    testCountSetBits("10", 1)
})
```
etc.